.agents/skills/release-doc/SKILL.md

```
@@ -0,0 +1,229 @@
+
+```bash
+# Try to extract version from release branch name
+# e.g. release/1.3.0 → 1.3.0, v1.3.0-rc1 → 1.3.0-rc1
+echo "<release_branch>" | sed -nE 's/.*([0-9]+\.[0-9]+\.[0-9]+[a-zA-Z0-9._-]*).*/\1/p'
+
+```bash
+# Release date = today in "1st January, 2026" format
+date +"%d %b, %Y" | sed 's/^0//' | sed 's/ 0/ /'
+```
+
+# Previous version = base tag (strip leading 'v' if present)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)